data/mods/maadowr/moves.ts

```diff
@@ -955,10 +958,13 @@ export const Moves: { [moveid: string]: ModdedMoveData } = {
 		pp: 10,
 		priority: 0,
 		flags: {protect: 1, mirror: 1, defrost: 1, metronome: 1},
-		secondary: {
-			chance: 100 if (this.field.isWeather('raindance', 'primordialsea')),
-			status: 'brn',
+		onSourceDamagingHit(damage, target, source, move) {
+			if (target.hasAbility('shielddust') || target.hasItem('covertcloak')) return;
+			if (this.field.isWeather('raindance', 'primordialsea')) {
```

Comment:
Hang on shouldn't it be this.field.isWeather(['raindance','primordialsea']) instead of this.field.isWeather('raindance','primordialsea')?

Reply:
Good catch. I've just changed it. Thanks!